python3-sys/Cargo.toml

```diff
@@ -37,6 +37,27 @@ default = ["python-3"]
 # so that the module can also be used with statically linked python interpreters.
 extension-module = [ ]
 
+# This feature implies default linking behavior.
+#
+# If not an extension module or on Windows, the crate will link against
+# pythonXY where XY is derived from the discovered Python version. The link
+# type will be static, shared, or framework depending on the discovered Python.
+#
+# The path to pythonXY from the discovered Python install may also be
+# added to the linker search path.
```

Comment:
Maybe here add to the comments that it's automatically activated (I can see that in `build.rs`) if other `link-mode` features aren't specified?

I suppose you didn't put it in `default` in order for the change to be transparently compatible for downstream crates that use `--no-default-features`, e.g, those that are for Python 2.7 ?

Reply:
Yes, I didn't put in in `default` because of implications for backwards compatibility or `--no-default-features`.